### PR TITLE
ViviCorporation.Vivi version 3.3.6

### PR DESCRIPTION
**Pull request by dmckayplc**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.2 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.2.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/89918)

### CODE REVIEW

File: manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+- silent
+- silentWithProgress
+InstallerSwitches:
+  Silent: /qn
+  SilentWithProgress: /qb
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
```
Comment (mdanish-kh) [inline manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml:20]:
```suggestion
  InstallerType: wix
```
InstallerType differences in new manifests can cause issues in upgrade scenario for the package. Although I'm almost sure setting this as `msi` here wouldn't be all that harmful; its better to keep it as its [previous value](https://github.com/microsoft/winget-pkgs/blob/200efa2a5e56552acffd8c3cfbba29a982667578/manifests/v/ViviCorporation/Vivi/3.2.0/ViviCorporation.Vivi.installer.yaml#L8).

File: manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  InstallerUrl: https://downloads.vivi.io/app/3.3.6/ViviSetup64.msi
+  InstallerSha256: AFD39113B01F6710D9F5AF3FC986E9DBCB6335DA173DF50596D3C4BE5A894965
+  ProductCode: '{A25E5188-3C67-4D72-91B0-E78083FF34CF}'
+- InstallerLocale: en-US
+  Architecture: neutral
+  InstallerType: msi
```
Comment (mdanish-kh) [inline manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml:26]:
```suggestion
  InstallerType: wix
```

File: manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,32 @@
...
+UpgradeBehavior: install
+Installers:
+- InstallerLocale: en-US
+  Architecture: x64
+  InstallerType: msi
+  InstallerUrl: https://downloads.vivi.io/app/3.3.6/ViviSetup64.msi
+  InstallerSha256: AFD39113B01F6710D9F5AF3FC986E9DBCB6335DA173DF50596D3C4BE5A894965
+  ProductCode: '{A25E5188-3C67-4D72-91B0-E78083FF34CF}'
+- InstallerLocale: en-US
+  Architecture: neutral
```
Comment (mdanish-kh) [inline manifests/v/ViviCorporation/Vivi/3.3.6/ViviCorporation.Vivi.installer.yaml:25]:
```suggestion
  Architecture: x86
```
I believe neutral covers all architectures (including arm/arm64); better to keep it as x86 here.

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Comment by mdanish-kh** [comment]
Please also sign the CLA following the steps outlined in [#89918 (comment)](https://github.com/microsoft/winget-pkgs/pull/89918?notification_referrer_id=NT_kwDOBUE-t7M0OTQ5NDExMTIzOjg4MTYxOTc1#issuecomment-1330036300)

**Comment by dmckayplc** [comment]
@microsoft-github-policy-service agree [company="{PLC}"]

**Comment by mdanish-kh** [comment]
@msftbot reset feedback

**Review by mdanish-kh — APPROVED** [review]

**Comment by ItzLevvie** [comment]
[Policy] Needs-CLA

**Pull request merged**